irsdb/metadata/models.py

```diff
@@ -1,4 +1,5 @@
 from django.db import models
+from django.contrib.postgres.fields import ArrayField
```

Comment:
i imagine you may not want to tie yourself to postgres. there is a portable version of jsonfield that could be used here.